ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/vlan-interfaces.yml

```diff
@@ -58,44 +92,122 @@ vlan_interfaces:
       interval: 500
       min_rx: 500
       multiplier: 5
+    # Test isis auth both sha
+    isis_enable: "EVPN_UNDERLAY"
+    isis_authentication:
+      both:
+        key_type: 0
+        key: password
+        mode: sha
+        sha:
+          key_id: 2
 
   - name: Vlan86
     description: SVI Description
     ip_address: 10.10.83.1/24
     ip_attached_host_route_export:
       enabled: true
       distance: 10
+    # Test isis auth both shared secret rx
+    isis_enable: "EVPN_UNDERLAY"
+    isis_authentication:
+      both:
+        mode: shared-secret
+        shared_secret:
+          profile: profile1
+          algorithm: sha-1
+        rx_disabled: true
 
   - name: Vlan87
     description: SVI Description
     shutdown: true
     ip_address: 10.10.87.1/24
     access_group_in: ACL_IN
     access_group_out: ACL_OUT
+    # Test isis auth both shared secret
+    isis_enable: "EVPN_UNDERLAY"
+    isis_authentication:
+      both:
+        mode: shared-secret
+        shared_secret:
+          profile: profile1
+          algorithm: sha-1
 
   - name: Vlan88
     description: SVI Description
     shutdown: true
     ip_address_virtual: 10.10.87.1/23
+    # Test isis auth both l1l2 md5 rx
+    isis_enable: "EVPN_UNDERLAY"
+    isis_authentication:
+      level_1:
```

Comment:
please update an example to have `different modes for level-1 and level-2` 
So, that it can be seen in the table also.

Reply:
Fixed.